patient-summary-llm/venv/Lib/site-packages/PIL/BdfFontFile.py

```
@@ -0,0 +1,133 @@
+
+        props = {}
+        comments = []
+
+        while True:
+            s = fp.readline()
+            if not s or s[:13] == b"ENDPROPERTIES":
+                break
+            i = s.find(b" ")
+            props[s[:i].decode("ascii")] = s[i + 1 : -1].decode("ascii")
+            if s[:i] in [b"COMMENT", b"COPYRIGHT"]:
+                if s.find(b"LogicalFontDescription") < 0:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)